wordpress-mobile/gutenberg-mobile
# Refactor ui strings using filters

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/25697

Implements changes from https://github.com/WordPress/gutenberg/pull/25697, following the recommendations from https://github.com/wordpress-mobile/gutenberg-mobile/pull/2610#pullrequestreview-496535098 to refactor the HOC as a filter.

![UBE_toggle](https://user-images.githubusercontent.com/9772967/94550214-a9d85400-0253-11eb-80ed-013b4418ed66.png)

**To test:**
- Run this PR on `gutenberg-mobile` iOS `npm run start:reset` `npm run start core ios`.
- Scroll to the bottom and press on the unsupported block UI.
- Check that the alert UI looks like in the screenshots.
- Toggle the Missing block UI Alert as shown on the screenshots and check the UI again.
- Do this three times and verify with the screenshots.

PR submission checklist:
